Template10 (Library)/Controls/HamburgerMenu.xaml.cs

```diff
@@ -410,8 +410,8 @@ internal void HighlightCorrectButton(Type pageType, object pageParam)
 
             // add parameter match
             buttons = buttons.Where(x => Equals(x.HamburgerButtonInfo.PageParameter, null) || Equals(x.HamburgerButtonInfo.PageParameter, pageParam));
-            var button = buttons.Select(x => x.HamburgerButtonInfo).FirstOrDefault();
-            Selected = button;
```

Comment:
Oops, we can't remove setting Selected as that's why this method exists, right? :)